apps/examples/testcase/ta_tc/media/itc/itc_media_mediaplayer.cpp

```diff
@@ -35,11 +35,17 @@ static int g_flag=0;
 
 static void SetUp(void)
 {
-	FILE *fp;
-	fp = fopen(file_path, "w");
-	TC_ASSERT_NEQ("fopen", fp, NULL);
-	TC_ASSERT_NEQ_CLEANUP("fputs", fputs(test_data, fp), EOF, fclose(fp));
-	TC_ASSERT_EQ("fclose", fclose(fp), OK);
+	FILE *fp = fopen(file_path, "w");
+
+        if (fp != NULL) {
+                int ret = fputs("dummydata", fp);
+                if (ret != (int)strlen("dummydata")) {
+                        printf("fail to fputs\n");
+                }
+                fclose(fp);
+        } else {
+                printf("fail to open %s, errno : %d\n", file_path, get_errno());
+        }
 	g_flag = 1;
```

Comment:
I think it is better to place this line inside the if loop. In the earlier case, since assert conditions are present, g_flag is not set. But in this case, g_flag will always be set.

Reply:
ok, will make changes as you suggested.